CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### 3.0.0-rc.0
+
+* The React component `AppContainer` is now `HotContainer`.

```

Comment:
We have to expose both AppContainer and HotContainer and deprecate AppContainer. The 3.0.0 beta is widely used and we should minimize breaking changes and take users by the hand.

Reply:
https://github.com/gaearon/react-hot-loader/pull/639

You're literally the person who deleted AppContainer?